packages/vm/core/root/internal.go

```
@@ -86,22 +86,25 @@ func GetChainInfo(state kv.KVStore) (*ChainInfo, error) {
 // GetFeeInfo is an internal utility function which returns fee info for the contract
 // It is called from within the 'root' contract as well as VMContext and viewcontext objects
 // It is not exposed to the sandbox
-func GetFeeInfo(state kv.KVStore, hname coretypes.Hname) (balance.Color, int64, int64, error) {
-	rec, err := FindContract(state, hname)
-	if err != nil {
-		// contract not found
-		return balance.Color{}, 0, 0, err
+func GetFeeInfo(state kv.KVStore, hname coretypes.Hname) (balance.Color, int64, int64) {
+	rec, _ := FindContract(state, hname)
```

Not sure why the error is ignored? Errors should never pass silently. At least `if err != nil { panic(err) }`?

---

returns nil if contract not found. The rest is default processing. No need to panic.

---

But the error could be a DB error? Schema inconsistency? Etc.